src/middlewared/middlewared/plugins/disk_/sync.py

```diff
@@ -79,116 +73,189 @@ async def sync_all(self, job):
         else:
             self.logger.info('Found %d disks', number_of_disks)
 
+        return number_of_disks
+
+    @private
```

Comment:
Both methods are great candidates for unit tests, for the future team members to have a list of examples of how we build, parse and map disk identifiers.

Reply:
Sure, no problem. You're talking about `dev_to_ident` and `ident_to_dev` right?

Reply:
Yes